build.sh

```diff
@@ -134,12 +140,102 @@ else
 	wait %1
 
 	# Run tests if activated
-	if [ $tests -eq 1 ]; then
+	if [ ${skip_tests} -eq 1 ] && [[ "${tests}" =~ .*"unittests".* ]]; then
 		echo "Running tests"
 		containerName=$(cat /proc/sys/kernel/random/uuid)
 		docker run --name $containerName $dockerArgs --rm \
 			"${buildImage}" \
 			/opt/gardenlinux/bin/garden-test &
 		wait %1
 	fi
+	if [ ${skip_tests} -eq 1 ] && [[ "${tests}" == *chroot* ]]; then
+		echo "Creating config file for chroot tests"
+		containerName=$(cat /proc/sys/kernel/random/uuid)
+		prefix="$(${thisDir}/bin/garden-feat --featureDir $featureDir --features "$features" --ignore "$disablefeatures" cname)-$arch-$version-$commitid"
+		mkdir -p ${thisDir}/config
+		cat > ${thisDir}/config/${containerName}.yaml << EOF
+chroot:
```

Comment:
In order to keep `build.sh` clean, I would probably move the configuration creation in a dedicated `bin/garden-*` script.